src/java.desktop/share/classes/sun/print/PrintJobDelegate.java

```diff
@@ -0,0 +1,1233 @@
+/*
+ * Copyright (c) 2000, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
should it be 2000? its a new class but its content is from 2000!!

Reply:
Not sure what you are saying.
You seem to be saying it should say 2000 .. but it does .. and yes it should ...
The filename doesn't matter the material is what matters.